jenkins/e2e-image/e2e-runner.sh

```diff
@@ -50,6 +50,7 @@ STAGE_KUBEMARK="KUBEMARK"
 
 : ${KUBE_GCS_RELEASE_BUCKET:="kubernetes-release"}
 : ${KUBE_GCS_DEV_RELEASE_BUCKET:="kubernetes-release-dev"}
+JENKINS_SOAK_PREFIX="gs://kubernetes-jenkins/soak/${JOB_NAME}"
```

Comment:
not sure if anyone else uses `e2e-runner.sh` for soak tests, but this probably needs to be overridable if they do.